### PR TITLE
Add rev A of macropad definition

### PR DESCRIPTION
**Pull request by ladyada**
tested workin!

### CODE REVIEW

File: ports/raspberrypi/boards/adafruit_macropad_rp2040/mpconfigboard.h [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#define MICROPY_HW_BOARD_NAME "Adafruit Macropad RP2040"
+#define MICROPY_HW_MCU_NAME "rp2040"
+
+#define MICROPY_HW_NEOPIXEL (&pin_GPIO16)
+
+#define DEFAULT_I2C_BUS_SCL (&pin_GPIO25)
+#define DEFAULT_I2C_BUS_SDA (&pin_GPIO24)
```
Comment (tannewt) [inline ports/raspberrypi/boards/adafruit_macropad_rp2040/mpconfigboard.h:7]: I don't see these pins in my copy. Have you pushed the latest files?

Comment (ladyada) [reply ports/raspberrypi/boards/adafruit_macropad_rp2040/mpconfigboard.h:7]: ok i just commited again.

**Comment by ladyada** [comment]
also tossed in a definition for the AREF pin which is technically an analog pin!

File: ports/raspberrypi/boards/adafruit_macropad_rp2040/pins.c [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    { MP_ROM_QSTR(MP_QSTR_KEY4), MP_ROM_PTR(&pin_GPIO4) },
+    { MP_ROM_QSTR(MP_QSTR_KEY5), MP_ROM_PTR(&pin_GPIO5) },
+    { MP_ROM_QSTR(MP_QSTR_KEY6), MP_ROM_PTR(&pin_GPIO6) },
+    { MP_ROM_QSTR(MP_QSTR_KEY7), MP_ROM_PTR(&pin_GPIO7) },
+    { MP_ROM_QSTR(MP_QSTR_KEY8), MP_ROM_PTR(&pin_GPIO8) },
+    { MP_ROM_QSTR(MP_QSTR_KEY9), MP_ROM_PTR(&pin_GPIO9) },
+    { MP_ROM_QSTR(MP_QSTR_KEY10), MP_ROM_PTR(&pin_GPIO10) },
+    { MP_ROM_QSTR(MP_QSTR_KEY11), MP_ROM_PTR(&pin_GPIO11) },
+    { MP_ROM_QSTR(MP_QSTR_KEY12), MP_ROM_PTR(&pin_GPIO12) },
+
```
Comment (tannewt) [inline ports/raspberrypi/boards/adafruit_macropad_rp2040/pins.c:16]: Want to add the buzzer too?

File: ports/raspberrypi/boards/adafruit_macropad_rp2040/pins.c [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    { MP_ROM_QSTR(MP_QSTR_ENCODER_SWITCH), MP_ROM_PTR(&pin_GPIO0) },
+
```
Comment (tannewt) [inline ports/raspberrypi/boards/adafruit_macropad_rp2040/pins.c:18]: I think you'll want D13 and an alias LED for it too.

**Review by tannewt — CHANGES_REQUESTED** [review]
Two more pins I think you'll want. Thanks for updating the files!

File: ports/raspberrypi/boards/adafruit_macropad_rp2040/pins.c (hunk 3 above)
Comment (ladyada) [reply ports/raspberrypi/boards/adafruit_macropad_rp2040/pins.c:18]: oh yah

File: ports/raspberrypi/boards/adafruit_macropad_rp2040/pins.c (hunk 2 above)
Comment (ladyada) [reply ports/raspberrypi/boards/adafruit_macropad_rp2040/pins.c:16]: i forgot it :D

**Review by tannewt — APPROVED** [review]
Looks good! Thank you!

**Pull request merged**